neuralmonkey/model/sequence.py

```diff
@@ -0,0 +1,192 @@
+from typing import List
+
+import tensorflow as tf
+from typeguard import check_argument_types
+
+from neuralmonkey.model.model_part import ModelPart, FeedDict
+from neuralmonkey.vocabulary import Vocabulary
+from neuralmonkey.decorators import tensor
+from neuralmonkey.dataset import Dataset
+
+
+class Sequence(ModelPart):
+
+    def __init__(self,
+                 name: str,
+                 max_length: int = None,
+                 save_checkpoint: str = None,
+                 load_checkpoint: str = None) -> None:
+        ModelPart.__init__(self, name, save_checkpoint, load_checkpoint)
+        check_argument_types()
+
+        self._max_length = max_length
+        if self._max_length is not None and self._max_length <= 0:
+            raise ValueError("Max sequence length must be a positive integer.")
+
+    @property
+    def data(self) -> tf.Tensor:
```

Comment:
Je otázka, jeslti by se to nemělo jmenovat `states` jako v těch enkodérech. Dovedu si představit, že bych klidně mohl chtít pustit rovnou sequence classifier na embeddingy, obzvlášť v nějakém multi-task setupu.